docs/source/include/optimizer.rst

```diff
@@ -47,14 +48,17 @@ in :eq:`eq:admm_prob`, corresponding to defining
   \mb{c} = \left( \begin{array}{c} 0 \\ 0 \\ 0 \\
               \vdots \end{array} \right) \;.
 
-In :class:`.ADMM`, :math:`f` is a :class:`.Functional`, typically a :class:`.Loss`, corresponding to the forward model of an imaging problem,
-and the :math:`g_i` are :class:`.Functional`, typically corresponding to a
-regularization term or constraint. Each of the :math:`g_i` must have a
-proximal operator defined. It is also possible to set ``f = None``, which corresponds to defining :math:`f = 0`, i.e. the zero function.
+In :class:`.ADMM`, :math:`f` is a :class:`.Functional`, typically a
+:class:`.Loss`, corresponding to the forward model of an imaging
+problem, and the :math:`g_i` are :class:`.Functional`, typically
+corresponding to a regularization term or constraint. Each of the
+:math:`g_i` must have a proximal operator defined. It is also possible
+to set ``f = None``, which corresponds to defining :math:`f = 0`,
+i.e. the zero function.
 
 
-Subproblem Solvers
-^^^^^^^^^^^^^^^^^^
+Sub-Problem Solvers
```

Comment:
Chicago manual of style disagrees, https://www.chicagomanualofstyle.org/book/ed17/part2/ch07/psec089.html Section 4.

Reply:
Fixed.